tests/dummy/app/components/au-example-scroller.hbs

```diff
@@ -0,0 +1,5 @@
+<div {{this.exampleScrollerModifier}} class="au-d-example-scroller">
```

Comment:
I don't think this is used / needed anymore now?

Reply:
I'm using this to test the positioning of the floating element (when using floating-ui), but I can remove it if you like and keep a copy of it locally 👍